src/libdrakvuf/libdrakvuf.h

```diff
@@ -472,6 +472,9 @@ bool drakvuf_get_current_thread_id(drakvuf_t drakvuf,
                                    drakvuf_trap_info_t* info,
                                    uint32_t* thread_id) NOEXCEPT;
 
+bool drakvuf_check_return_context(drakvuf_t drakvuf, drakvuf_trap_info_t* info,
```

Comment:
I would like if you have added a comment describing what this function is intended to do.

Reply:
I added a comment. My English is bad, but I tried.